packages/react-slider/src/components/Slider/useSliderStyles.ts

```diff
@@ -188,13 +193,75 @@ const useTrackStyles = makeStyles({
   }),
 });
 
+/**
+ * Styles for the mark slot
+ */
+const useMarksWrapperStyles = makeStyles({
+  marksWrapper: theme => ({
+    position: 'relative',
+    display: 'grid',
+    outline: 'none',
+    zIndex: '1',
+    whiteSpace: 'nowrap',
+    [`& .${markClassName}`]: {
+      // TODO: theme neutralStrokeOnBrand
```

Comment:
Is this supposed to be addressed now or later?

Reply:
Later once the token is added. I just updated the comment to mention that.